Sources/StreamVideo/OpenApi/generated/Models/CallDurationReport.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public final class CallDurationReport: @unchecked Sendable, Codable, JSONEncodable, Hashable {
+    
+    public var histogram: [ReportByHistogramBucket]
```

Comment:
Is it easy to change all properties in our models to be `let` rather than `var`? Classes shouldn't really have `var` properties (and especially public) ones as the same reference can be found in multiple places and if we change the value in one of them all places will be updated. (`var` in structs are different in the sense that structs are value types and we cannot have the same reference in 2 different places)